### PR TITLE
Update carpenter build push yaml

### PR DESCRIPTION
**Pull request by BATMAN-JD**
## Changes introduced with this PR

Updating carpenter build and push yaml files to build latest tag for main branch
Will continue to build branch and short hash tag for other branches.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: .github/workflows/carpenter_build.yaml [hunk 1]
```diff
@@ -2,25 +2,42 @@ name: Carpentry
 on:
   pull_request:
 env:
-  IMAGE_NAME: arcaflow-plugin-kubeconfig
+  IMAGE_NAME: arcaflow-plugin-template-python
```
Comment (dustinblack) [inline .github/workflows/carpenter_build.yaml]: Oops -- Need to update the name.

File: .github/workflows/carpenter_push.yaml [hunk 2]
```diff
@@ -4,22 +4,39 @@ on:
     branches:
       - "**"
 env:
-  IMAGE_NAME: arcaflow-plugin-kubeconfig
+  IMAGE_NAME: arcaflow-plugin-template-python
```
Comment (dustinblack) [inline .github/workflows/carpenter_push.yaml]: Oops

**Review by dustinblack — CHANGES_REQUESTED** [review]

File: .github/workflows/carpenter_build.yaml (hunk 1 above)
Comment (BATMAN-JD) [reply .github/workflows/carpenter_build.yaml]: darn you are right. not the first time I have done that mistake from copy pasta. Thank you

File: .github/workflows/carpenter_push.yaml (hunk 2 above)
Comment (BATMAN-JD) [reply .github/workflows/carpenter_push.yaml]: updated

**Review by dustinblack — APPROVED** [review]

**Pull request merged**